homework-submission/react-hyf/react-hyf-client/src/components/week3/Homework3.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState, useEffect } from 'react';
+import UserInfo from './component/UserInfo';
+
+import './Homework3.css';
+
+function Homework3() {
+  const [users, setUsers] = useState([]);
+  const [isLoaded, setIsLoaded] = useState(true);
+  const [index, setIndex] = useState(0);
+  const [error, setError] = useState(false);
+
+  const genRandom = index => {
+    window.addEventListener('keyup', e => {
```

Comment:
Currently the page also scrolls down due to pressing space, can you prevent that?